module/equal/equal.go

```diff
@@ -28,7 +29,7 @@ func (m *EqualModule) Name() string { return equalName }
 
 func (m *EqualModule) Execute(targets map[string]pgs.File, pkgs map[string]pgs.Package) []pgs.Artifact {
 	// Process file-level templates
-	tpl := templates.Template(m.Parameters())
+	tpl := templates.Template(m.Parameters(), hash.Register) // TODO: create equal.Register
```

Comment:
Does this still work, seems like it would be registering this module twice?

Reply:
it is exactly the same behavior as before, just refactored

Reply:
i think the answer is that using `equal_all` will currently produce invalid code (which was the existing behavior)